roles/rabbitmq/tasks/main.yml

```diff
@@ -15,3 +15,13 @@
     enabled: yes
     state: started
   tags: rabbitmq
+
+- name: Add RabbitMQ plugins
+  become: yes
+  rabbitmq_plugin:
+    name: "{{ item }}"
+    state: enabled
+    prefix: "{{ is_redhat6 | ternary(rmq_redhat6_prefix, omit) }}"
+  with_items: "{{ rmq_plugins }}"
+  when: rmq_plugins
+  tags: rabbitmq
```

Comment:
Do we need to restart rabbitmq once the plugin is added?

Reply:
I don't think so:

> The rabbitmq-plugins comand will enable or disable plugins by updating the plugin configuration file. It will then contact the running server to tell it to start or stop plugins as needed. You can use the  -n option to specify a different node, or use  --offline to only change the file.
https://www.rabbitmq.com/plugins.html

I read that to mean that a running rabbitmq both loads/unloads the plugin without a restart (as long as the --offline flag isn't used. The rabbitmq_plugin module doesn't have a way to include the offline flag, so we shouldn't ever need a restart when adding/removing plugins.